src/protocols/aeternity/libs/AeternityAdapter.ts

```diff
@@ -145,6 +146,8 @@ export class AeternityAdapter extends BaseProtocolAdapter {
       name: this.coinName,
       symbol: this.coinSymbol,
       convertedBalance,
+      // This price is in coin per token
```

Comment:
This comment is not necessary as the interface has this property already commented. I suggest to use DocBlock there to have IDE tooltips with the comment.